config/locales/en.yml

```diff
@@ -298,6 +298,7 @@ en:
     confirm_deletion: Are you sure you want to delete this assignment?
     delete: Remove and Delete
     editors: Assigned to
+    change_sandbox_url: Change Sandbox
```

Comment:
On further reflection, I think this is better as lower case: 'change sandbox', so that feels a little more differentiated from the sandbox link itself.

Reply:
lower case for button also ?